src/pages/user-email-collection.jsx

```diff
@@ -1,22 +1,26 @@
-import React, { Component } from 'react';
-import Subtitle from '../components/subtitle/subtitle';
-import Privacy from '../containers/privacy/privacy'
-import EmailForm from '../containers/email-form/email-form';
+import React, { Component } from "react";
+import Subtitle from "../components/subtitle/Subtitle";
+import Privacy from "../containers/privacy/Privacy";
+import EmailForm from "../containers/email-form/email-form";
 
 class UserEmailCollection extends Component {
-   state = {
+    state = {};
 
-   };
+    render() {
```

Comment:
In app.js you render this component, so in order to get the props to the EmailForm, you need to pass them through this component too. Basically App.js > UserEmailCollection > EmailForm

so in order for the props to reach EmailForm they need to go though UserEmailCollection